### PR TITLE
fix: small refactor and adding userstory infrastructure

### PR DESCRIPTION
**Pull request by carlospatinos**
UserStory Not fully tested. The rest manually tested. 
I see a lot of code duplication with controllers since they are so simple, maybe later they will pay off
We still need to work on testing. 
Should we have a testing db infrastructure? mock?

### CODE REVIEW

File: packages/scrum-client/src/constants/commonFunctions.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+const getValueFromLocalStorage = (itemId, key) => {
+  if (localStorage && itemId && key) {
```
Comment (doncesarts) [inline packages/scrum-client/src/constants/commonFunctions.js]: there is a util package for functions , the folder constants is for constants only

File: packages/scrum-server/src/api/components/planningSession/service.js [hunk 2]
```diff
@@ -22,14 +20,13 @@ const save = async req => {
...
-const find = async req => {
-  const planningRoomId = req.params.id;
-  if (!planningRoomId || !ObjectId.isValid(planningRoomId)) {
-    throw Error(i18n.__('apiPlanningSessionInvalidId'));
+const find = async objectId => {
+  if (!objectId || !ObjectId.isValid(objectId)) {
+    throw Error(i18n.__('serviceInvalidId'));
   }
-  const planningSession = await PlanningSession.findOne({ _id: planningRoomId });
+  const planningSession = await PlanningSession.findOne({ _id: objectId });
```
Comment (doncesarts) [inline packages/scrum-server/src/api/components/planningSession/service.js]: why objetId is better than planningRoomId?

File: packages/scrum-server/src/api/components/user/service.js [hunk 3]
```diff
@@ -45,9 +45,10 @@ const signUp = async req => {
   }
 };
 
+// TODO remove the req 
 const localAuth = async req => {
   const token = req.cookies.auth;
-
+  console.log('token', token);
```
Comment (doncesarts) [inline packages/scrum-server/src/api/components/user/service.js]: is this console expected?

File: packages/scrum-server/src/api/components/userStory/model.js [hunk 4]
```diff
@@ -0,0 +1,28 @@
+const mongoose = require('mongoose');
+
```
Comment (doncesarts) [inline packages/scrum-server/src/api/components/userStory/model.js:2]: good

File: packages/scrum-server/src/api/components/userStory/service.js [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+  }
+};
+
+const find = async objectId => {
+  if (!objectId || !ObjectId.isValid(objectId)) {
+    throw Error(i18n.__('serviceInvalidId'));
+  }
+  const userStory = await UserStory.findOne({ _id: objectId });
+  if (!userStory) {
+    throw Error(i18n.__('serviceFindError'));
```
Comment (doncesarts) [inline packages/scrum-server/src/api/components/userStory/service.js:26]: Shall we create error constants ?

**Review by doncesarts — APPROVED** [review]

**Comment by doncesarts** [comment]
In overall it is a good pr with some improvements that could be done

File: packages/scrum-server/src/api/components/planningSession/service.js (hunk 2 above)
Comment (carlospatinos) [reply packages/scrum-server/src/api/components/planningSession/service.js]: My bad, i was removing the request in the service and changed the name unintentionally, sorted in the next push

File: packages/scrum-server/src/api/components/user/service.js (hunk 3 above)
Comment (carlospatinos) [reply packages/scrum-server/src/api/components/user/service.js]: Kind of, ill remove it for now, we need better ways to troubleshoot errors

File: packages/scrum-server/src/api/components/userStory/service.js (hunk 5 above)
Comment (carlospatinos) [reply packages/scrum-server/src/api/components/userStory/service.js:26]: Lets discuss that, i think yes makes sense, but will that have to be dynamic? for the i18n?

**Pull request merged**